configure-load-base-split.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Load Base Split
+summary: Learn the feature of Load Base Split.
+category: how-to
+---
+
+# Load Base Split
+
+Load Base Split is a new feature introduced in TiDB 4.0. It aims to solve the hotspot issue caused by unbalanced access between Regions, such as full table scans for small tables.
+
+## Scenarios
+
+In TiDB, it is easy to create hotspots when the load is concentrated on certain nodes. PD tries to schedule the Hot Regions so that they are distributed as evenly as possible across all nodes for better performance.
```

Comment:
```suggestion
In TiDB, it is easy to generate hotspots when the load is concentrated on certain nodes. PD tries to schedule the hot Regions so that they are distributed as evenly as possible across all nodes for better performance.
```